haml/_header.html.haml

```diff
@@ -8,11 +8,11 @@
   .cads-grid-container
     .cads-grid-row
       %nav.cads-skip-links
-        %a.cads-skip-links__link{ href: "#cads-navigation" }
-          = t("cads.header.skip_to_nav")
-        %a.cads-skip-links__link{ href: main_content_anchor_link }
+        %a.cads-skip-links__link{ data: { test_id: "skip_link" }, href: "#cads-navigation" }
```

Comment:
I might be missing something but you don't seem to be using these ids now?  If not just remove them.

Reply:
Oh yes. I am not.